includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -364,6 +364,8 @@ public function __construct(
 
 		// Update the current request logged_in cookie after a guest user is created to avoid nonce inconsistencies.
 		add_action( 'set_logged_in_cookie', [ $this, 'set_cookie_on_current_request' ] );
+
+		add_action( 'admin_init', [ $this, 'maybe_enable_upe_feature_flag' ] );
```

Comment:
It might be nice to colocate this in the note itself, like we discussed in the team meeting. In the main comment, I added the diff that has been working for me.